templates/public/securedrop.html

```diff
@@ -61,7 +60,7 @@ <h4 class='gu-heading'>How to access the Guardian's SecureDrop service</h4>
             For security reasons, we advise you, especially if you are uploading documents, not to use your home or work network, but instead to use a public Wi-Fi network in an  area where your screen is not visible to security cameras. Alternately, you can start up your computer from a USB key loaded with the Tails secure operating system, which is available at <a href="https://tails.boum.org/">https://tails.boum.org</a> and includes the Tor web browser.
           </li>
           <li class="gu-text">
-            Once you launch the Tor browser, copy and paste the URL <span class="onion-url">{{ securedrop_url }}</span> into the Tor address bar. When the page loads, you will find further instructions on how to submit files and messages to the Guardian.
+            Once you launch the Tor browser, copy and paste the URL <span class="onion-url">{{ securedrop_url }}</span> or <span class="onion-url">{{ securedrop_url_human }}</span> into the Tor address bar. When the page loads, you will find further instructions on how to submit files and messages to the Guardian.
```

Comment:
Could

<h4 class='gu-heading'>How to access the Guardian's SecureDrop service</h4>

use a smart apostrophe pls? ('the Guardian’s')